### ISSUE TITLE
Debian Installation Instructions Lacking

### ISSUE THREAD

**Issue by DarioS** [BodenmillerGroup/histoCAT#68]
In Assets, I see histoCAT_1.75_Debian.install but there are no instructions about what to do with that file. I would know what to do if it is a DEB file but the suffix `install` is a mystery. Please add a section to the installation guide about Linux installs. Also, a NEWS.md file specifying changes in each version would be informative.

**Comment by DenisSch** [comment]
@DarioS Good point! I will try to create a DEB file for the newest version. Which Debian distribution are you using? Should I just do it on 10.6?

**Comment by DarioS** [comment]
That'll do me just fine.

```
$ lsb_release -d
Description:    Debian GNU/Linux 10 (buster)
```

**Comment by iivario** [comment]
Trying to install histoCAT in Debian linux 10. When running histoCAT_1.75_Debian.install to install histoCAT, the installation stops with an error. The Matlab R2014b is installed.   

`$ ./histoCAT_1.75_Debian.install  

Failed to download the MCR Installer from http://www.mathworks.com/pi_install_agent_pi_R2014b_glnxa64Error : Unsupported protocol
Could not source the install agent
Installation failed.
`
